test/__snapshots__/test.js.snap

```diff
@@ -523,47 +545,255 @@ const Impl = require(\\"../implementations/Factory.js\\");
+                context: \\"Failed to set the 'unforgeableAttr' property on 'Global': The provided value\\"
+              });
+
+              obj[impl][\\"unforgeableAttr\\"] = V;
+            },
+            get length() {
+              if (!this || !module.exports.is(this)) {
+                throw new TypeError(\\"Illegal invocation\\");
+              }
+
+              return obj[impl][\\"length\\"];
+            },
+            set length(V) {
+              if (!this || !module.exports.is(this)) {
+                throw new TypeError(\\"Illegal invocation\\");
+              }
+
+              V = conversions[\\"unsigned long\\"](V, {
+                context: \\"Failed to set the 'length' property on 'Global': The provided value\\"
+              });
+
+              obj[impl][\\"length\\"] = V;
+            },
+            [Symbol.iterator]: Array.prototype[Symbol.iterator],
+            keys: Array.prototype.keys,
+            values: Array.prototype[Symbol.iterator],
+            entries: Array.prototype.entries,
+            forEach: Array.prototype.forEach
+          },
+          {
+            unforgeableOp: {
+              configurable: false,
+              enumerable: true,
+              writable: false
+            },
+            unforgeableAttr: {
+              configurable: false,
+              enumerable: true
+            },
+            [Symbol.iterator]: {
+              configurable: true,
+              enumerable: false,
+              writable: true
+            }
+          }
+        )
+      )
+    );
+  },
+  setup(obj, constructorArgs, privateData) {
+    if (!privateData) privateData = {};
+
+    privateData.wrapper = obj;
+
+    this._internalSetup(obj);
+    Object.defineProperty(obj, impl, {
+      value: new Impl.implementation(constructorArgs, privateData),
+      writable: false,
+      enumerable: false,
+      configurable: true
+    });
+
+    obj[impl][utils.wrapperSymbol] = obj;
+    if (Impl.init) {
+      Impl.init(obj[impl], privateData);
+    }
+    return obj;
+  },
+  interface: Global,
+  expose: {
+    Window: { Global }
+  }
+}; // iface
+module.exports = iface;
+
+const Impl = require(\\"../implementations/Global.js\\");
+"
+`;
+
+exports[`LegacyArrayClass.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const impl = utils.implSymbol;
 
-Object.defineProperty(LegacyArrayClass.prototype, \\"length\\", {
-  get() {
+class LegacyArrayClass {
+  constructor() {
+    throw new TypeError(\\"Illegal constructor\\");
+  }
+
+  get length() {
     if (!this || !module.exports.is(this)) {
       throw new TypeError(\\"Illegal invocation\\");
     }
 
     return this[impl][\\"length\\"];
+  }
+}
+Object.setPrototypeOf(LegacyArrayClass.prototype, Array.prototype);
+Object.defineProperties(LegacyArrayClass.prototype, {
+  length: {
+    configurable: true,
```

Comment:
configurable is true by default; no need to set it explicitly, I don't think

Reply:
> in general I'd love to reduce the redundant property descriptor values.

I could work on this, but it just seems to be better form to have everything explicitly written out rather than depending some flags on default behaviors while modifying others.

Reply:
I just get a bit worried about code size, but maybe it's fine... I like the succinctness of only changing what's actually different, I guess. Shrug.

Reply:
You're right... the delta-only output does look very nice. PR updated.